### PR TITLE
RD-3755 Add step-by-step guide for agent install_method: provided

### PR DESCRIPTION
**Pull request by glukhman**

### CODE REVIEW

File: content/install_maintain/agents/installation.md [hunk 1]
```diff
@@ -107,6 +107,53 @@ install the agent package, but only configures and starts the agent
...
+1. Untar the archive to a folder with the same as the node instance. 
+   ("container_nwxqiu" in the above example):
+   ```shell
+   tar xzfv centos-core-agent.tar.gz --strip=1 -C ~centos/container_nwxqiu
+   ```
+1. Download the configuration script using the like from step 4:
+   ```shell
+   wget --no-check-certificate --user admin --password admin https://10.239.0.207:53333/resources/cloudify_agent/3631bbc3-fc37-4a9f-9db3-81adb2d8e182.sh
+   ```
+1. The script doesn't copy the SSL certificate, so just do it manually &mdash;
```
Comment (geokala) [inline content/install_maintain/agents/installation.md:143]: Shouldn't that be &emdash?

Comment (geokala) [reply content/install_maintain/agents/installation.md:143]: (I'm not certain of that)

File: content/install_maintain/agents/installation.md [hunk 2]
```diff
@@ -107,6 +107,53 @@ install the agent package, but only configures and starts the agent
...
+   pem` in the VM:
+   ```shell
+   openssl s_client -connect 10.239.0.207:53229 </dev/null 2>/dev/null |   openssl x509 -outform PEM > centos/container_nwxqiu/cloudify/ssl/cloudify_internal_cert.pem
+   ```
+1. Run the downloaded script:
+   ```shell
+   sudo ./3631bbc3-fc37-4a9f-9db3-81adb2d8e182.sh
+   ```
+   
+The agent has now joined to the {{< param product_name >}} manager.
```
Comment (geokala) [inline content/install_maintain/agents/installation.md:155]: Should we add a note that we recommend this process be automated by some means or is that obvious enough?

File: content/install_maintain/agents/installation.md (hunk 1 above)
Comment (glukhman) [reply content/install_maintain/agents/installation.md:143]: nope, Hugo renders it correctly

**Review by geokala — APPROVED** [review]

File: content/install_maintain/agents/installation.md [hunk 3]
```diff
@@ -107,6 +107,54 @@ install the agent package, but only configures and starts the agent
 daemon. This script needs to be downloaded and executed manually for
 every agent.
 
+#### Provided agent step-by-step-guide
+
+1. Prepare a Virtual Machine or Image with custom a agent. Either Prepare
```
Comment (alexmolev) [inline content/install_maintain/agents/installation.md:112]:
```suggestion
1. Prepare a Virtual Machine or Image with a custom agent. Either Prepare 
```

File: content/install_maintain/agents/installation.md [hunk 4]
```diff
@@ -107,6 +107,54 @@ install the agent package, but only configures and starts the agent
 daemon. This script needs to be downloaded and executed manually for
 every agent.
 
+#### Provided agent step-by-step-guide
+
+1. Prepare a Virtual Machine or Image with custom a agent. Either Prepare 
+   a custom agent or just upload an agent archive from https://<manager 
+   IP>:53229/packages/agents/<br>
+   For versions 5.x use link https://<managerIP>:53333/resources/cloudify_agent/
```
Comment (alexmolev) [inline content/install_maintain/agents/installation.md:115]: Why version 5.x? is the link different for other versions?

**Review by alexmolev — CHANGES_REQUESTED** [review]

**Review by alexmolev — APPROVED** [review]

**Pull request merged**